web/src/components/forms/upload-schema-form.tsx

```diff
@@ -22,6 +22,28 @@ type Props = {
   defaultNamespace?: string;
 };
 
+type CombinedErrorMessageProps = {
+  errors: FieldErrors<FromFileInputs>;
+};
+
+const CombinedErrorMessage = (props: CombinedErrorMessageProps) => {
```

Comment:
Second (third?) time I've seen this... do you think its worth abstracting out into a separate component?

Reply:
fixed